Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1500 @@ label mas_wrs_twitch:
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_solitaire:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Oh, you like playing with cards? Nice!"
+           "You know, [player], I wouldn't mind playing a card game with you\nwhen I cross over. That'd be nice, don't you agree?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_solitaire')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_tiktok",
+            category=['tiktok'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_tiktok:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Hmmm, there's a lot of creative stuff being made here.\nDo you make content on here, [player]?", 
+           "If I ever got on here, I bet you'd follow me in a heartbeat. Ehehe~"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_tiktok')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_whatsapp",
+            category=['whatsapp'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_whatsapp:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Chatting with your friends, [player]? You should introduce me to them sometime!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_whatsapp')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_paint3d",
+            category=['paint3d'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_paint3d:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Doing some image editing, player?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_paint3d')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_cortana",
```

Comment:
As I recall, Cortana isn't its own window. this basically would never work. Can be removed.